tests/foreign_object/test_agnostic_order_trimjoin.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
```

Comment:
No need for these lines -- only include the coding preamble if we have non-ASCII characters in the file.